src/main/java/de/doubleslash/keeptime/controller/Controller.java

```diff
@@ -237,29 +236,35 @@ public void setComment(final String notes) {
     * Calculate todays seconds counted as work
     */
    public long calcTodaysWorkSeconds() {
+      final List<Work> workItems = new ArrayList<>();
 
-      return model.getPastWorkItems().stream().filter(work -> {
-         final Project project = work.getProject();
-         // find up to date reference to project
-         final Optional<Project> optionalProject = model.getAllProjects().stream()
-               .filter(p -> p.getId() == project.getId()).findAny();
-         if (optionalProject.isPresent()) {
-            return optionalProject.get().isWork();
+      for (final Work w : model.getPastWorkItems()) {
```

Comment:
please reuse the old logic with the id equals. Not sure why i did it like this, but it was working :) (u can still use the for each)

Reply:
Ok I understand why :)